server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface.js

```diff
@@ -145,6 +181,44 @@ class HttpClientInterface extends layerProtocol {
         return httpClientUuid != undefined;
     }
 
+    /**
+     * @description This function checks if application exists other than old release and new release
+     * @param {String} applicationName : name of the application.
+     * @param {String} releaseNumber : release number of the application.
+     * @returns {Promise<boolean>} true|false
+     **/
+    static async isApplicationExistsExcludingOldReleaseAndNewRelease(applicationName, releaseNumber) {
+        let httpClientUuid = await HttpClientInterface.getHttpClientUuidExcludingOldReleaseAndNewRelease(applicationName, releaseNumber);
+        return httpClientUuid != undefined;
+    }
```

Comment:
Thus function is just a middle-man, please remove it and use directly the content.